pkg/process/exec.go

```diff
@@ -9,6 +9,9 @@ import (
 	"os"
 
 	"github.com/spf13/cobra"
+
+	// We use a blank import here to get the side effects from the init function in version
+	_ "storj.io/storj/internal/version"
```

Comment:
Shouldn't be necessary.

Reply:
Thats used to verify the Version Info on build time, that were handed over with the ldfags.
I am not sure what the best way is to do that.